lib/rules/jsx-no-bind.js

```diff
@@ -111,6 +114,11 @@ module.exports = {
       return null;
     }
 
+    /**
+     * @param {string | number} violationType
+     * @param {string} variableName
```

Comment:
@ljharb May you review the changes that I have made here?

Reply:
I think this PR needs to be rebased onto latest main, but in general it seems good :-)